node/translate_text/src/index.js

```diff
@@ -0,0 +1,29 @@
+const googleProvider = require('./google_provider')
+const azureProvider = require('./azure_provider')
+const awsProvider = require('./aws_provider');
+
+module.exports = async function(req, res) {
+    const {
+      provider,
+      text
+    } = JSON.parse(req.payload);
+
+    let translatedText = "";
+
+    if(provider==='google'){
+      translatedText = await googleProvider(req.payload).catch((e)=>console.log(e));
```

Comment:
On error, please makes sure to return an error response as detailed in the issue.